docs/ZAP-on-Windows.md

```diff
@@ -0,0 +1,111 @@
+# Install ZAP from Source on Windows
+
+## Prerequisite Package & Settings
+
+**1. Windows Powershell:** 
+
+In the desktop search bar, input "Windows Powershell" and run as administrator
+
+**2. Clone Repo:**
+```
+git clone https://github.com/project-chip/zap
+```
+If having file permission issue, clone it under \Users\<user_name> directory, do not put it under Desktop
+
+**3. Chocolatey:**
+
+install from https://chocolatey.org/install
+check if installed properly:
+```
+choco -v
+```
+Install pkgconfiglite package:
+```
+choco install pkgconfiglite
+```
+
+**4. Install Node:**
+```
+choco install nodejs-lts
+```
+*the version has to be 18 to pass version check test, after install, check with "node -v"
+
+*if you install Node already, and fail some tests similar to "cannot find Node", reinstall Node with chocolatey again
+
+## Run ZAP App
+
+**1. Install node modules:**
+```
+npm install
```

Comment:
@YanchengZhou 
For ubuntu/mac/fedora, there are simplified install files such as following and they are used by the Github CI job a well.
Can you create a windows version and update the release.yml to include it in the CI? 
That way, your work will not be an one-off thing and will be tested continuously. 

src-script/install-packages-fedora
src-script/install-packages-osx
src-script/install-packages-ubuntu